### PR TITLE
updated/fixed de_DE.lang

### PR DESCRIPTION
**Pull request by Scoaex**
fixed typos, improved some translations that made no sense (or 2 different translations for 1 thing in achievements/blocks),
removed the "-" after ME (not needed, there is no in other lang files aswell),...

### CODE REVIEW

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
 tile.appliedenergistics2.charger.name=Ladegerät
-tile.appliedenergistics2.chest.name=ME-Truhe
+tile.appliedenergistics2.chest.name=ME Truhe
 tile.appliedenergistics2.condenser.name=Materiekondensator
-tile.appliedenergistics2.controller.name=ME-Controller
+tile.appliedenergistics2.controller.name=ME Controller
 tile.appliedenergistics2.crank.name=Holzkurbel
 tile.appliedenergistics2.creative_energy_cell.name=Kreative Energiezelle
-tile.appliedenergistics2.dense_energy_cell.name=Dichte Energiezelle
+tile.appliedenergistics2.dense_energy_cell.name=Komprimierte Energiezelle
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Wrong. This is about high density and not compression.

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 2]
```diff
@@ -25,12 +25,12 @@ tile.appliedenergistics2.quartz_glass.name=Quarzglas
 tile.appliedenergistics2.quartz_vibrant_glass.name=Polarisiertes Quarzglas
 tile.appliedenergistics2.quartz_pillar.name=Certusquarzsäule
 tile.appliedenergistics2.quartz_fixture.name=Geladene Quarzleuchte
-tile.appliedenergistics2.light_detector.name=Quarzlichtsensor
-tile.appliedenergistics2.spatial_io_port.name=IO-Raumport
+tile.appliedenergistics2.light_detector.name=Quarz Lichtsensor
+tile.appliedenergistics2.spatial_io_port.name=IO Port
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: It is still an spatial port, not a normal IO Port

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 3]
```diff
@@ -25,12 +25,12 @@ tile.appliedenergistics2.quartz_glass.name=Quarzglas
...
 tile.appliedenergistics2.spatial_pylon.name=Raumpylon
-tile.appliedenergistics2.tiny_tnt.name=Mini-TNT
+tile.appliedenergistics2.tiny_tnt.name=Mini TNT
 tile.appliedenergistics2.vibration_chamber.name=Vibrationskammer
-tile.appliedenergistics2.wireless_access_point.name=Drahtloser ME-Zugangspunkt
+tile.appliedenergistics2.wireless_access_point.name=ME Drahtloser Zugangspunkt
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: The order is somewhat wrong as this is a access point with the additional property of being wireless. Could also be a tethered one.

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 4]
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
...
 tile.appliedenergistics2.sky_compass.name=Meteoritenkompass
 tile.appliedenergistics2.crafting_monitor.name=Fertigungsmonitor
-tile.appliedenergistics2.crafting_storage_1k.name=1k-Fertigungsspeicher
-tile.appliedenergistics2.crafting_storage_4k.name=4k-Fertigungsspeicher
-tile.appliedenergistics2.crafting_storage_16k.name=16k-Fertigungsspeicher
-tile.appliedenergistics2.crafting_storage_64k.name=64k-Fertigungsspeicher
-tile.appliedenergistics2.crafting_accelerator.name=Fertigungs-Prozessoreinheit
-tile.appliedenergistics2.crafting_unit.name=Fertigungseinheit
-tile.appliedenergistics2.molecular_assembler.name=Molekularassembler
+tile.appliedenergistics2.crafting_storage_1k.name=1k Speicherzelle
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Plain wrong. Still crafting storage not normal ones.

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 5]
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
...
+tile.appliedenergistics2.crafting_storage_4k.name=4k Speicherzelle
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Same

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 6]
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
...
+tile.appliedenergistics2.crafting_storage_16k.name=16k Speicherzelle
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Same

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 7]
```diff
@@ -43,13 +43,13 @@ tile.appliedenergistics2.sky_stone_chest.name=Himmelssteintruhe
...
+tile.appliedenergistics2.crafting_storage_64k.name=64k Speicherzelle
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Same

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 8]
```diff
@@ -293,33 +295,33 @@ gui.tooltips.appliedenergistics2.MoveWhenWorkIsDone=Auswerfen wenn fertig.
 gui.tooltips.appliedenergistics2.MoveWhenFull=Auswerfen wenn voll.
 gui.tooltips.appliedenergistics2.Disabled=Deaktiviert
 gui.tooltips.appliedenergistics2.Enable=Aktiviert
-gui.tooltips.appliedenergistics2.Blocking=Nicht fertigen, wenn Zielinventar Items enthält.
+gui.tooltips.appliedenergistics2.Blocking=Keine Fertigungsitems einfügen während das Zielinventar Items enthält.
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: "Fertigungsitems" is not really a word. At least it should be "Gegenstand" or whatever the official translation for item is used by minecraft.

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 9]
```diff
@@ -417,45 +419,45 @@ item.appliedenergistics2.material.Singularity.name=Singularität
...
-item.appliedenergistics2.multi_part.CableCovered.name=Verdecktes ME-Kabel
+item.appliedenergistics2.multi_part.CableCovered.name=Verdecktes ME Kabel
 item.appliedenergistics2.multi_part.CableGlass.name=ME-Glaskabel
-item.appliedenergistics2.multi_part.CableSmart.name=Schlaues ME-Kabel
-item.appliedenergistics2.multi_part.CableDense.name=Dichtes ME-Kabel
-item.appliedenergistics2.multi_part.StorageMonitor.name=ME-Speichermonitor
-item.appliedenergistics2.multi_part.ConversionMonitor.name=ME-Konversionsmonitor
-item.appliedenergistics2.multi_part.CraftingTerminal.name=ME-Fertigungskonsole
+item.appliedenergistics2.multi_part.CableSmart.name=Schlaues ME Kabel
+item.appliedenergistics2.multi_part.CableDense.name=Dickes ME Kabel
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Also about density not size.

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang [hunk 10]
```diff
@@ -487,10 +489,10 @@ item.appliedenergistics2.matter_cannon.name=Materiekanone
 item.appliedenergistics2.memory_card.name=Speicherkarte
 item.appliedenergistics2.color_applicator.name=Farbapplikator
 item.appliedenergistics2.wireless_terminal.name=Drahtlose Zugangskonsole
-item.appliedenergistics2.biometric_card.name=Biometriekarte
+item.appliedenergistics2.biometric_card.name=Biometrische Karte
 
-item.appliedenergistics2.debug_card.name=Dev.Debug-Karte
-item.appliedenergistics2.debug_replicator_card.name=Dev.Replikator-Karte
+item.appiedenergistics2.debug_card.name=Dev.Debug Karte
```
Comment (yueh) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: `appliedenergistics2`

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang (hunk 1 above)
Comment (Scoaex) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: True, but the 1:1 translation makes no sense here in my opinion cause "Dicht" doesnt  really fit for that, thats why i looked for something better

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang (hunk 4 above)
Comment (Scoaex) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Messed it up here with the cells, my bad

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang (hunk 9 above)
Comment (Scoaex) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: Same with Energy Cell, used "Dick" here cause it made more sense here
I´ll change it back to "Dicht" until someone comes up with a better translation

**Comment by Scoaex** [comment]
Fixed my typos and that i accidently mixed up crafting storage and the storage cells.
Also reverted to the old "Dense" translation as you said, maybe someone comes up with something better since the 1:1 translation doesnt really fit here in my opinion.

File: src/main/resources/assets/appliedenergistics2/lang/de_DE.lang (hunk 9 above)
Comment (M4lik) [inline src/main/resources/assets/appliedenergistics2/lang/de_DE.lang]: I would also replace "Schlaues ME Kabel" with "Itelligentes ME Kabel"...

**Comment by bolli24** [comment]
I would use "kompakt" instead of "dicht". In my opinion this fits the best.

**Pull request closed without merging**